core/models.py

```diff
@@ -413,6 +413,10 @@ class DataProcess(ReferenceDataEnum):
         blank=True,
     )
 
+    is_using_automation = models.CharField(
+        choices=YesNoDontKnowChoices.choices, null=True, default=False
```

Comment:
The default should be `None`.

```suggestion
        choices=YesNoDontKnowChoices.choices, null=True, default=None
```